src/Parser.js

```diff
@@ -787,11 +787,8 @@ Parser.prototype.parseSizeGroup = function(optional) {
         number: +(match[1] + match[2]), // sign + magnitude, cast to number
         unit: match[3],
     };
-    if (data.unit !== "em" && data.unit !== "ex" && data.unit !== "mu") {
-        throw new ParseError("Invalid unit: '" + data.unit + "'", res);
-    }
```

Comment:
We should continue to throw here, but check if `data.unit` is one of the keys in `ptPerUnit` (assuming we add `em` and `mu` to it).